pkg/telemetry/statsworker.go

```diff
@@ -80,11 +80,6 @@ func (s *StatsWorker) Flush() {
 	ts := timestamppb.Now()
 
 	s.lock.Lock()
-	if !s.closedAt.IsZero() {
```

Comment:
due to close timing, sometimes publish events could come in *after* participant left is called. In those cases we would have missed events that aren't flushed.

it's harmless to flush additional events here even after close.